doc/mkdocs.yml

```diff
@@ -34,6 +34,7 @@ nav:
       - Service mode:
           - GitHub webhook configuration: user-guide/service/webhook.md
           - Run service using pre-built Docker image: user-guide/service/docker.md
+          - Quick Start Guide: user-guide/service/quick-start-guide.md
```

Comment:
Incorrect place. Check line 31 and replace Getting Started with Quick Start Guide.